clients/fides-js/src/components/fides.css

```diff
@@ -0,0 +1,426 @@
+  bottom: 0;
+  left: 0;
+}
+
+div#fides-banner-container.fides-banner-bottom.fides-banner-hidden {
+  transform: translateY(150%);
+}
+
+div#fides-banner-container.fides-banner-top {
+  top: 0;
+  left: 0;
+}
+
+div#fides-banner-container.fides-banner-top.fides-banner-hidden {
+  transform: translateY(-150%);
+}
+
+/* Responsive banner */
+@media screen and (min-width: 48em) {
+  div#fides-banner {
+    width: 75%;
+    border-radius: var(--fides-overlay-component-border-radius);
+    border: 1px solid var(--fides-overlay-primary-color);
+  }
+  div#fides-banner-container.fides-banner-bottom {
+    bottom: var(--fides-overlay-banner-offset);
+  }
+  div#fides-banner-container.fides-banner-top {
+    top: var(--fides-overlay-banner-offset);
+  }
+}
+
+@media only screen and (min-width: 80em) {
+  div#fides-banner {
+    width: 60%;
+  }
+}
+
+div#fides-banner-title {
+  font-size: var(--fides-overlay-font-size-title);
+  font-weight: 600;
+  margin-top: 0.2em;
+  min-width: 33%;
+  flex: 1;
+  color: var(--fides-overlay-title-font-color);
+}
+
+div#fides-banner-description {
+  margin-top: 0.75em;
+  line-height: 1.2em;
+  min-width: 33%;
+  flex: 1;
+}
+
+div#fides-banner-buttons {
+  margin-top: 0.5em;
+}
+
+span.fides-banner-buttons-right {
+  float: right;
+}
+
+button.fides-banner-button {
+  font-size: var(--fides-overlay-font-size-buttons);
+  display: inline-block;
+  flex: auto;
+  cursor: pointer;
+  text-align: center;
+  margin: 0;
+  margin-top: 0.25em;
+  margin-right: 0.5em;
+  padding: var(--fides-overlay-button-padding);
+  background: var(--fides-overlay-primary-button-background-color);
+  color: var(--fides-overlay-primary-button-text-color);
+  border: 1px solid;
+
+  font-family: inherit;
+  line-height: 1.15;
+  text-decoration: none;
+}
+
+button.fides-banner-button:hover {
+  background: var(--fides-overlay-primary-button-background-hover-color);
+}
+
+button.fides-banner-button.fides-banner-button-primary {
+  background: var(--fides-overlay-primary-button-background-color);
+  color: var(--fides-overlay-primary-button-text-color);
+  border: none;
+}
+
+button.fides-banner-button.fides-banner-button-primary:hover {
+  background: var(--fides-overlay-primary-button-background-hover-color);
+}
+
+button.fides-banner-button.fides-banner-button-secondary {
+  background: var(--fides-overlay-secondary-button-background-color);
+  color: var(--fides-overlay-secondary-button-text-color);
+  border: 1px solid var(--fides-overlay-primary-button-background-color);
+}
+
+button.fides-banner-button.fides-banner-button-secondary:hover {
+  background: var(--fides-overlay-secondary-button-background-hover-color);
+}
+
+button.fides-banner-button.fides-banner-button-tertiary {
+  background: none;
+  border: none;
+  padding: 0;
+  color: var(--fides-overlay-link-font-color);
+  text-decoration: underline;
+  cursor: pointer;
+  line-height: 2em;
+}
+
+/** Modal */
+div.modal-content {
```

Comment:
I think for many of these we want to prefix with `fides-`, just in case the hosted site also has a classname `modal-content`, or `toggle`, etc